### ISSUE TITLE
add random labels to utils.get_data()

### ISSUE THREAD

**Issue by charlesmartin14** [CalculatedContent/ww-phys_theory#1]
I suggest adding, in the middle of the method:

 ```
 if random_labels:
    print("generating random labels with seed {}".format(seed))
    np.random.seed(seed)

    probability_of_random = 1.0
    labels = np.array(train_dataset.targets) 
    mask = np.random.rand(len(labels)) <= probability_of_random #create mask of length labels, where entries drawn from [0,1].
    rnd_labels = np.random.choice(10, mask.sum())               #create random labels 1-10 of length of mask
    labels[mask] = rnd_labels
    labels = [int(x) for x in labels]
    train_dataset.targets = labels                              #assign new random labels to dataset
    np.savetxt("random_labels.txt", labels)

```

**Comment by johnshin86** [comment]
Added code.
